src/ApparentHorizons/Tags.hpp

```diff
@@ -238,3 +212,32 @@ using compute_items_tags =
                LaplacianRadius<Frame>, NormalOneForm<Frame>, Tangents<Frame>>;
 
 }  // namespace StrahlkorperTags
+
+/// \ingroup SurfacesGroup
+/// Holds tags and ComputeItems associated with a `::Strahlkorper` that
+/// also need a metric.
+namespace StrahlkorperGrTags {
```

Comment:
To be consistent with how we name other `Tags`-namespaces:
`StrahlkorperGr::Tags`